config/services.yml

```diff
@@ -11,119 +11,123 @@ services:
 
   fop.console.cache.clear.command:
     class: FOP\Console\Commands\Cache\CacheClear
-    tags: [ console.command ]
+    tags: [console.command]
```

Comment:
Could you please change only the new command you add and not all the commands declarations ?